manifests/init.pp

```diff
@@ -113,8 +110,46 @@
   $wsgi_threads      = $sentry::params::wsgi_threads,
 ) inherits ::sentry::params {
 
+  class { '::sentry::setup':
```

Comment:
This isn't proving quite as straightforward as I expected.  More work is needed.  I'd rather not delay this merge just for this tweak.